pkg/yurthub/storage/store.go

```diff
@@ -16,33 +16,115 @@ limitations under the License.
 
 package storage
 
-import (
-	"errors"
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+type ClusterInfoKey struct {
+	ClusterInfoType
+	UrlPath string
+}
+
+type ClusterInfoType string
+
+const (
+	Version          ClusterInfoType = "version"
+	APIsInfo         ClusterInfoType = "apis"
+	APIResourcesInfo ClusterInfoType = "api-resources"
+	Unknown          ClusterInfoType = "unknown"
 )
 
-// ErrStorageAccessConflict is an error for accessing key conflict
-var ErrStorageAccessConflict = errors.New("specified key is under accessing")
+// Store is an interface for caching data into store
+type Store interface {
+	// Name will return the name of this store.
+	Name() string
+	clusterInfoRelatedInterface
+	objectRelatedInterface
+	componentRelatedInterface
```

Comment:
The definition of interfaces should be match with proposal. 
for example: clusterInfoRelatedInterface --> clusterInfoHandler

Reply:
fixed